src/main/java/controller/LeaveRequestController.java

```diff
@@ -9,13 +9,13 @@
 import java.time.LocalDate;
 
 public class LeaveRequestController {
-    private final LeaveRequestInputBoundary userInput;
+    final LeaveRequestInputBoundary userInput;
 
     public LeaveRequestController(LeaveRequestInputBoundary userInput) {
         this.userInput = userInput;
     }
 
-    LeaveRequestResponseModel create(User user, java.lang.String message, LeaveType leaveType,
```

Comment:
Could you briefly explain why you change the type of the message from String to java.lang.String?

Reply:
I  messed up my entities and InteliiJ automatically changed all of it to java.lang.String, I am still trying to find all it and change it back to String. I don't think there is any difference between them.